spreadsheet/sharedstrings.go

```diff
@@ -57,6 +58,15 @@ func (s SharedStrings) GetString(id int) (string, error) {
 	si := s.x.Si[id]
 	if si.T != nil {
 		return *si.T, nil
+	} else if l := len(si.R); l > 0 {
+		if l == 1 {
```

Comment:
I think it would be better to remove this specific check. It seems like a premature optimization to me. strings.Join will still work with the slice of 1-length. Doing this check we may loose the speed because of the CPU branch-prediction, this specific case requires benchmarks and some statistics on the different docs. I think it would be easier just to remove this and simplify the code